tests/conftest.py

```diff
@@ -23,51 +23,67 @@ def path_test_file(cwd):
 
 
 @pytest.fixture(scope='session')
-def serpent_depcode(cwd):
-    """SerpentDepcode object for unit tests"""
+def serpent_runtime(cwd, tmpdir_factory):
+    """SaltProc objects for Serpent unit tests"""
     saltproc_input = str(cwd / 'serpent_data' / 'tap_input.json')
     _, _, _, object_input = read_main_input(saltproc_input)
     depcode = _create_depcode_object(object_input[0])
     depcode.runtime_inputfile = str(cwd / 'serpent_data' / 'tap_reference')
+    output_dir = str(depcode.output_path).split('/')[-1]
+    depcode.output_path = tmpdir_factory.mktemp(f'serpent_{output_dir}')
+
+    simulation = _create_simulation_object(object_input[1], depcode, 1, 1)
+        #db_path=str(
+        #    cwd /
+        #    'serpent_data' /
+        #    'tap_reference_db.h5'))
+
+    reactor = _create_reactor_object(object_input[2])
```

Comment:
Consider adding extra variables i.e. `simulation_object = object_input[1]`, that way its clearer what is being pulled from `object_input`

Reply:
I agree

Reply:
okay